UoaWDCC/younite-web
# 062 migrate to layout

## Context

We want to have a consistent page layout for each page.

Closes #62 

## What Changed?

- Removed `<Header />` and `<Footer />` components from each individual page
- Added footer and header to `layout.tsx`
- Adjusted `layout.tsx`
  - Now contains `<main />` (which is now a column [flexbox](https://www.joshwcomeau.com/css/interactive-guide-to-flexbox/))
  - Header is now [positioned absolutely](https://www.w3schools.com/css/css_positioning.asp)
  - Footer now stays at the bottom of the page if the page is too small
  - Adjusted pages to look like before*

## How To Review


- Start up the app and check nothing's broken on each of the pages!
- Check out `layout.tsx` and the home page `page.tsx` pages first
- Review code commit-wise

## Notes

*New tailwind styles: `mt-header` and `pt-header`
